core/kazoo_documents-1.0.0/src/kz_account.erl

```diff
@@ -156,12 +156,13 @@ is_superduper_admin(JObj) ->
 allow_number_additions(JObj) ->
     wh_json:is_true(?ALLOW_NUMBER_ADDITIONS, JObj).
 
--spec fetch(ne_binary()) -> {'ok', doc()} |
+-spec fetch(api_binary()) -> {'ok', doc()} |
                             {'error', _}.
+fetch('undefined') ->
```

Comment:
It's actually to reflect behaviour that used to exist before. I mean the switch from open_cache_doc to fetch removed this "dont crash on undefined, but return error" behaviour.